packages/components/src/components/buttons/TooltipButton/TooltipButton.tsx

```diff
@@ -3,17 +3,17 @@ import styled, { useTheme } from 'styled-components';
 import { Icon } from '../../assets/Icon/Icon';
 import { Tooltip } from '../../Tooltip/Tooltip';
 import { Button, ButtonProps } from '../Button/Button';
+import { spacingsPx } from '@trezor/theme';
 
 const StyledButton = styled(Button)`
     position: relative;
-    padding-left: 32px;
-    padding-right: 32px;
+    padding-inline: ${spacingsPx.xxl};
```

Comment:
I didn't know that this exists, good to know 👍

Reply:
Yes, thanks, but beware, padding-inline is not 100% the same as using padding-left or right. Orientation of the screen matters when using it. However if both values are the same it does not matter and we can safely use it like this.